internal/npm_package/packager.js

```diff
@@ -17,20 +17,37 @@
 const fs = require('fs');
 const path = require('path');
 
+const EXCLUDE_REPLACEMENTS = [
```

Comment:
This fixes a bug in the packager where it was making replacements in the .tar.gz file included in the package.

Reply:
Moved to another PR